packages/ketcher-core/src/application/ketcher.ts

```diff
@@ -29,6 +29,22 @@ import assert from 'assert'
 import { EventEmitter } from 'events'
 import { runAsyncAction } from 'utilities'
 
+const Format = {
```

Comment:
I propose to add type for this object. It can be something like:
```
const Format: {
  [key in SupportedFormat]: SupportedFormat
}
```
This way you don't need to write 
```
Format.SomeFormat as SupportedFormat
```

Reply:
Fixed, pls check, if it's a good variant